scripts/postselected_boson_sampling_hist.py

```diff
@@ -0,0 +1,65 @@
+#
```

Comment:
Suggest putting the word `histogram` at the end of the file, follows convention from other files, and at first I didn't understand what `hist` meant. Also, some file docstring explaining briefly what's going on would help.